dietpi/misc/dietpi-nordvpn

```diff
@@ -155,34 +198,19 @@ _EOF_
 	# TARGETMENUID=0
 	Menu_Main(){
 
-		Update_Wan_Ip
-
-		local text_status="Status:\n - WAN IP: $WAN_IP\n"
```

Comment:
Perhaps this "Status:" line is needless?
Just have:
```
WAN IP      : <IP>
State       : <Not configured|Connected|Disconnected>
Traffic     : Sent = ... | Received = ... # If connected
Subscription: https://... # If not connected or configured 
```

Reply:
Agreed!